src/sdk/pynni/nni/platform/standalone.py

```diff
@@ -14,7 +14,9 @@
     'send_metric',
 ]
 
-init_standalone_logger()
+if trial_env_vars.NNI_PLATFORM is None:
+    init_standalone_logger()
```

Comment:
this file is for standalone mode, `trial_env_vars.NNI_PLATFORM` must be `None`, right?

Reply:
My understanding is yes,  we can see `trial_env_vars.NNI_PLATFORM is None` is used to check standalone mode at other places:
https://github.com/microsoft/nni/blob/master/src/sdk/pynni/nni/trial.py#L111
https://github.com/microsoft/nni/blob/master/src/sdk/pynni/nni/platform/__init__.py#L6

Reply:
Good catch, the problem is on environment variables, NNI_PLATFORM is an env var for trial not for dispatcher,  I will fix it in another PR.